src/components/engine-connector/engine-connector.js

```diff
@@ -59,10 +59,14 @@ class EngineConnector extends BaseComponent {
       this.props.updateDuration(this.player.duration);
     });
 
+    this.player.addEventListener(this.player.Event.SOURCE_SELECTED, () => {
+      // TODO: change when player will expose the player.poster at this stage
+      this.props.updatePlayerPoster(this.player.config.metadata.poster);
```

Comment:
@dvh91 `player.poster` is already exposed@https://github.com/kaltura/playkit-js/blob/master/src/player.js#L632

Reply:
@OrenMe not at the stage of `SOURCE_SELECTED`.